squidadm/code-maintenance.sh

```diff
@@ -0,0 +1,107 @@
+#!/bin/sh
```

Comment:
Please do not add a directory called "squidadm" (in this or any other PR). I can suggest "maintenance", but I do not insist on any specific name. There are probably other good names that describe what kind of files this directory will be aggregating.